src/Console/EnumMakeCommand.php

```diff
@@ -8,78 +8,17 @@
 
 class EnumMakeCommand extends GeneratorCommand
 {
-    /**
-     * The name and signature of the console command.
-     *
-     * @var string
-     */
```

Comment:
I removed all of them, the comment vs. code ratio was extremely for mostly code taking advantage of the parent class anyway => tidied this up